src/pm/hydra/lib/utils/alloc.c

```diff
@@ -382,6 +382,7 @@ HYD_status HYDU_create_proxy_list(int count, struct HYD_exec *exec_list, struct
         }
     }
 
+    *min_node_id_p = min_node_id;
```

Comment:
so the `min_node_id` is the lowest id for that particular group of processes/proxies? just confirming my understanding.

Reply:
Yes